### DISCUSSION TITLE
Which file should I get to update from 0.0.5 to .6 on Linux?

### DISCUSSION THREAD

**Discussion by TiagoTiago** [carson-katri/dream-textures#231] [Q&A]
###

**Comment by carson-katri** [comment]
You need to install manually using the instructions under “Contributing” on linux

**Reply by TiagoTiago** [reply]
> You need to install manually using the instructions under “Contributing” on linux

Can't I just get a zip file like with the previous version?

**Reply by carson-katri** [reply]
You do download the source code zip then follow the instructions to manually install dependencies.

**Reply by TiagoTiago** [reply]
Hm, I guess I must be misremembering it then; I thought I had just installed the previous version thru a zip file like a regular Blender addon...

**Comment by Anyeos** [comment]
What i did (and worked) was:

1. Clone the 0.0.7 version with "`git clone https://github.com/carson-katri/dream-textures.git`"
2. Wait to finish and cd that directory
3. Execute: `git submodule update --init --recursive`
4. Wait to finish
5. Copy the complete recently cloned directory structure ("dream-textures") under the "addons" folder of Blender (mine is in ~/.config/blender/3.3/scripts/addons).
6. Copy (or symlink) the stable diffusion model file "model.ckpt" (4 GB or 8 GB version) under "stable_diffusion/models/ldm/stable-diffusion-v1" subfolder (create the last one if it does not exists).
7. Run Blender go to Addons, search for it and enable it.

A lot of steps but it works.
